src/flags/sorting.rs

```diff
@@ -156,13 +136,24 @@ impl Default for SortOrder {
 }
 
 /// The flag showing where to place directories.
-#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+#[derive(Clone, Debug, Copy, PartialEq, Eq, Deserialize)]
+#[serde(rename_all = "kebab-case")]
 pub enum DirGrouping {
     None,
     First,
     Last,
 }
 
+impl DirGrouping {
+    fn from_str(value: &str) -> Option<Self> {
+        match value {
+            "first" => Some(Self::First),
+            "last" => Some(Self::Last),
+            "none" => Some(Self::None),
+            _ => None,
```

Comment:
If the other `from_str` methods either print warnings or panic, this should as well. Especially since the previous implementation panicked.